kernel-open/nvidia/linux_nvswitch.c

```diff
@@ -711,7 +711,7 @@ _nvswitch_os_free
 
     if (is_vmalloc_addr(ptr))
     {
-        vfree(ptr);
+        vfree(ptr, -1ull);
```

Comment:
@francescolavra I don't know how to call vfree here. Please guide me.

Maybe it's like this?
```
 hdr = get_hdr(p);
 vfree(hdr, hdr->alloc_size);
```

Reply:
Should I export the `uvm_vmalloc_hdr_t` structure for reference in kernel-open/nvidia/linux_nvswitch.c?